Model/Data/Product/DataValidator.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Itonomy\Katanapim\Model\Data\Product;
+
+use Itonomy\Katanapim\Model\Data\Product\DataValidator\ValidatorInterface;
+
+/**
+ * Class DataValidator
+ */
+class DataValidator
+{
+    /**
+     * @var ValidatorInterface[]
+     */
+    private array $defaultValidators;
+
+    /**
+     * @var ValidatorInterface[]
+     */
+    private array $scopeValidators;
+
+    /**
+     * @var array
```

Comment:
Why are the first two clearly stated ValidatorInterface[], but this one only array?